src/bundle/Resources/views/content/content_view_fields.html.twig

```diff
@@ -1,32 +1,44 @@
 <section class="ez-view-rawcontentview">
+    {% set preview_is_collapsed = ez_user_settings['collapse_content_preview'] %}
     <div class="ez-raw-content-title d-flex justify-content-between mb-3">
         <h2>
-            {{ 'tab.view.content'|trans()|desc('Content') }}
+            <a data-toggle="collapse" href=".ez-content-preview-collapse" {% if preview_is_collapsed %}class="collapsed"{% endif %}>
+                {{ 'tab.view.preview'|trans()|desc('Preview') }}
+
+                <svg class="ez-icon ez-icon--collapsed">
+                    <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#caret-next"></use>
+                </svg>
+                <svg class="ez-icon ez-icon--expanded">
```

Comment:
You can use only one icon and do `rotate()` on it. We would be able to add transition animation on rotate and be consistent in whole application.